### ISSUE TITLE
Discriminator Loss

### ISSUE THREAD

**Issue by duynn912** [leftthomas/SRGAN#29]
In your code, d_loss = 1 - real_out + fake_out and I cannot understand this line code. Can you explain it?

**Comment by leftthomas** [comment]
@duynn912 you should read some blogs about GAN to figure out how they act and you will see the formula.

**Comment by micklexqg** [comment]
@leftthomas, are you sure d_loss = 1 - real_out + fake_out is right?
